### PR TITLE
Update ba.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
add youtube regular as i dont know how to fit 2 lives stream from the streamlinker proxy

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: channels/ba.m3u [hunk 1]
```diff
@@ -9,13 +9,17 @@ https://bhrtstream.bhtelecom.ba/hls15/bhrtportal.m3u8
...
+#EXTINF:-1 tvg-id="ntvic.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/Y4UJEfg.png" group-title="",NTV IC
+https://iptv.dc-team.com/query?url=https://www.twitch.tv/ntvic
 #EXTINF:-1 tvg-id="RTVBPK.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/g639yyT.jpg" group-title="",RTV BPK (720p)
 https://webtvstream.bhtelecom.ba/gorazde_cam2.m3u8
 #EXTINF:-1 tvg-id="RTVHB.com" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/EE8C8rR.png" group-title="",RTV HB (576p)
 https://prd-hometv-live-open.spectar.tv/ERO_1_083/playlist.m3u8
 #EXTINF:-1 tvg-id="Televizija5.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://i.imgur.com/hdwjkYa.png" group-title="",Televizija 5 (240p)
 https://balkanmedia.dynu.net/hls/tv5web.m3u8
-#EXTINF:-1 tvg-id="tvslon.ba" tvg-country="BA" tvg-language="Bosnian" tvg-logo="https://www.rtvslon.ba/wp-content/uploads/2020/07/slon-extra-logo-mali-300x277.jpg" group-title="",TV Slon Extra (1080p) [Offline]
-http://31.47.0.130:8082
```
Comment (Dum4G) [inline channels/ba.m3u:19]: image
Connects and plays as fast as if it was my local stream

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ba.m3u (hunk 1 above)
Comment (dct-infra) [reply channels/ba.m3u:19]: same here.

**Comment by Sphinxroot** [comment]
added back

File: channels/ba.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/ba.m3u:19]: Same here.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**